packages/api/src/routes/trade.ts

```diff
@@ -33,6 +33,11 @@ const trades: FastifyPluginAsync = async (fastify, opts): Promise<void> => {
       if (PLAYER_MINT_TIMESTAMP && isTimeToMint())
         return reply.status(403).send(new Error(`Trade period is over.`))
 
+      // Cooldown parameter is only allowed in tests
```

Comment:
Why adding a small value to `TRADE_DURATION_MILLIS` and `TRADE_COOLDOWN_MILLIS` in .env.test doesn't work here?

Reply:
Because it breaks the other tests that need a different cooldown